bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/target/env/AsterismType.java

```diff
@@ -13,42 +13,72 @@
 import java.util.function.Supplier;
 
 public enum AsterismType implements DisplayableSpType {
-    Single("single", "Single Target", None.instance(), Asterism$.MODULE$::createSingleAsterism),
 
-    GhostSingleTarget("ghostSingleTarget", "Single Target",
-            new Some<>(AsterismConverters.GhostSingleTargetConverter$.MODULE$),
-            GhostAsterism$.MODULE$::createEmptySingleTargetAsterism),
+    Single(
+        "single",
+        "Single Target",
+        ResolutionMode.Standard,
+        None.instance()
+    ),
 
-    GhostDualTarget("ghostDualTarget", "Dual Target",
-            new Some<>(AsterismConverters.GhostDualTargetConverter$.MODULE$),
-            GhostAsterism$.MODULE$::createEmptyDualTargetAsterism),
+    GhostSingleTarget(
+        "ghostSingleTarget",
+         "Single Target",
+         ResolutionMode.GhostStandard,
+         new Some<>(AsterismConverters.GhostSingleTargetConverter$.MODULE$)
+    ),
 
-    GhostTargetPlusSky("ghostTargetPlusSky", "SRIFU1 Target, SRIFU2 Sky",
-            new Some<>(AsterismConverters.GhostTargetPlusSkyConverter$.MODULE$),
-            GhostAsterism$.MODULE$::createEmptyTargetPlusSkyAsterism),
+    GhostDualTarget(
+        "ghostDualTarget",
+        "Dual Target",
+        ResolutionMode.GhostStandard,
+        new Some<>(AsterismConverters.GhostDualTargetConverter$.MODULE$)
+    ),
 
-    GhostSkyPlusTarget("ghostSkyPlusTarget", "SRIFU1 Sky, SRIFU2 Target",
-            new Some<>(AsterismConverters.GhostSkyPlusTargetConverter$.MODULE$),
-            GhostAsterism$.MODULE$::createEmptySkyPlusTargetAsterism),
+    GhostTargetPlusSky(
+        "ghostTargetPlusSky",
+        "SRIFU1 Target, SRIFU2 Sky",
+        ResolutionMode.GhostStandard,
+        new Some<>(AsterismConverters.GhostTargetPlusSkyConverter$.MODULE$)
+    ),
 
-    GhostHighResolutionTargetPlusSky("ghostHRTargetPlusSky", "HRIFU Target, Sky",
-            new Some<>(AsterismConverters.GhostHRTargetPlusSkyConverter$.MODULE$),
-            GhostAsterism$.MODULE$::createEmptyTargetPlusSkyAsterism)
+    GhostSkyPlusTarget(
+        "ghostSkyPlusTarget",
+        "SRIFU1 Sky, SRIFU2 Target",
+        ResolutionMode.GhostStandard,
+        new Some<>(AsterismConverters.GhostSkyPlusTargetConverter$.MODULE$)
+    ),
 
+    GhostHighResolutionTargetPlusSky(
+        "ghostHRTargetPlusSky",
+        "HRIFU Target, Sky",
+        ResolutionMode.GhostHigh,
+        new Some<>(AsterismConverters.GhostHRTargetPlusSkyConverter$.MODULE$)
+    ),
+
+    GhostHighResolutionTargetPlusSkyPrv(
+        "ghostHRTargetPlusSkyPrv",
+        "HRIFU Target, Sky (PRV)",
```

Comment:
The "(PRV)" suffix here might not be appreciated.  I added it so that I could verify that the asterism type was correct when the "Precision Radial Velocity" resolution mode was selected.  I think it'd be fine to remove "(PRV)" though and have both of the HR asterism types be just "HRIU Target, Sky".